src/components/basic/input/Text.tsx

```diff
@@ -22,7 +22,7 @@ export const TextInput = View<Props>(({ placeholder, value, onChange, disabled =
   )
 })
 
-const TextInputComponent = styled.input`
```

Comment:
I actually reverted this change because it makes the height of this component inconsistent with other UI components (e.g., button). Do you find severe usability issue with this?

Reply:
> I actually reverted this change because it makes the height of this component inconsistent with other UI components (e.g., button). Do you find severe usability issue with this?

I feel like some contents are hard to follow because the line doesn't wrap and users have to drag the mouse all the way around the edge highlighting elements for those cases which is cumbersome. However, since consistency is important we can leave this because of the work around.